pipelines/shared/Jenkinsfile

```diff
@@ -95,10 +94,9 @@ pipeline {
                     intention.startAction("login")
                     def vaultToken = intention.provisionToken("login", CONFIG_ROLE_ID)
                     def vault = new Vault(vaultToken)
-                    def cdCreds = vault.read('groups/data/appdelivery/jenkins-isss-cdua')
-                    env.PODMAN_CD_USER = cdCreds['username_lowercase']
-                    env.PODMAN_CD_PASS = cdCreds['password']
-                    vault.readToObject(env.ARTIFACTORY_VAULT_PATH, env, keyTransform: { key -> 'REGISTRY_' + key.toUpperCase() })
+                    vault.readToObject("apps/data/prod/jenkins/jenkins-apps/jdk_cacerts_pass", env, keyTransform: { key -> key })
```

Comment:
passing folder will convert all secrets with original key to env.key
vault.readToObject("apps/data/prod/jenkins/jenkins-apps", env)